apps/parsley/cypress/integration/sectioning.ts

```diff
@@ -0,0 +1,80 @@
+describe("Sectioning", () => {
+  beforeEach(() => {
+    cy.visit(`${logLink}?shareLine=0`);
+    cy.clickToggle("sections-toggle", true, "log-viewing");
+  });
+  it("Toggling the sections options displays and hides sections", () => {
+    // Check that sections is toggled
+    cy.toggleDetailsPanel(true);
+    cy.get("button[data-cy='log-viewing-tab']").click();
+    cy.dataCy("sections-toggle").should("have.attr", "aria-checked", "true");
+    cy.toggleDetailsPanel(false);
+    // Assert sections are visible
+    cy.dataCy("section-header").should("be.visible");
+    // Untoggle sections and assert they are hidden
+    cy.clickToggle("sections-toggle", false, "log-viewing");
+    cy.dataCy("section-header").should("not.exist");
+  });
+  it("Clicking 'Open all subsections' opens all subsections", () => {
```

Comment:
This checks that the button updates, can we check the sections themselves are open?